### ISSUE TITLE
High Level Overview

### ISSUE THREAD

**Issue by tudormunteanu** [oanaOM/auction-house#4]
Tell me the work flow of the app, in plain English, as an high level overview of the steps that your code executes. From the start, to the end results.

**Comment by oanaOM** [comment]
The work flow would be:
- Open and read the input file
- Get a list of all auctions from the file
- Close the file
- Iterate through the all auctions list, identify which one is a BID or a SELL and create a list for each category.
- Iterate through all items in sell list and the bids for each one and mark if an items was SOLD or UNSOLD. At this stage we’ll calculate the total number of valid bids made per item.
- An item is SOLD if the bid arrives after the auction starting time and before the closing time, his price is garter than starting price or any previous valid bids.
- Store the final status of each item in a new list along with some additional details
- Create a new file called ‘output.txt’ where we’ll store the final results
- Write to that file the final status compiled 2 steps before
- Close the file
